### PR TITLE
LazyDroid: add tool

### LINKED ISSUE #1717: [Tool request] wscript
https://github.com/mrpapercut/wscript

### PR DESCRIPTION
**Pull request by Edu4rdSHL**
Fix #1717

### CODE REVIEW

**Comment by Edu4rdSHL** [comment]
Reopening.

**Comment by noptrix** [comment]
thx @Edu4rdSHL

File: packages/lazydroid/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+pkgname=lazydroid
+_pkgname=LazyDroid
+pkgver=25.0f559ec
+pkgrel=1
+pkgdesc='Tool written as a bash script to facilitate some aspects of an Android Assessment'
+arch=('any')
+groups=('blackarch' 'blackarch-binary' 'blackarch-automation' 'blackarch-mobile')
+url='https://github.com/nccgroup/LazyDroid'
+license=('Apache')
+depends=()
```
Comment (FFY00) [inline packages/lazydroid/PKGBUILD]: @noptix, can you remove this?

File: packages/lazydroid/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+makedepends=('wget')
```
Comment (FFY00) [inline packages/lazydroid/PKGBUILD]: This should be in `depends`.

Comment (noptrix) [reply packages/lazydroid/PKGBUILD]: yep, was goin to remove/fix it.

**Pull request merged**